Tools/TestWebKitAPI/Tests/IPC/StreamConnectionTests.cpp

```diff
@@ -390,7 +390,7 @@ TEST_P(StreamMessageTest, SendAsyncReplyCancel)
     auto cleanup = localReferenceBarrier();
 
     std::atomic<bool> waiting;
-    BinarySemaphore workQueueWait;
+    static BinarySemaphore workQueueWait;
```

Comment:
Same problem exists with the `waiting` variable.

This could be instead:

```
TEST_P(StreamMessageTest, SendAsyncReplyCancel)
{
    auto cleanup = localReferenceBarrier();

```